webapp/channels/src/components/file_preview_modal/file_preview_modal.tsx

```diff
@@ -172,13 +189,22 @@ export default class FilePreviewModal extends React.PureComponent<Props, State>
         }
         const fileType = Utils.getFileType(fileInfo.extension);
 
-        if (fileType === FileTypes.IMAGE && isFileInfo(fileInfo)) {
+        // Check if this is an image URL, even if the extension is not recognized
+        const isImage = fileType === FileTypes.IMAGE ||
+                       (!isFileInfo(fileInfo) && this.isImageUrl(fileInfo.link));
```

Comment:
I stumbled over `!isFileInfo` and then using `fileInfo.link`, but I'm guessing we reuse this structure to capture information about embedded (and not upload) images?

Reply:
yes, exactly. The second part is specifically for checking external image URLs that could not had been recognized extension or might be proxied through a image third party service.